### PR TITLE
Fix typos and allow new Json naming conventions.

### PR DESCRIPTION
**Pull request by danielbatchford**
- Fix typos in axis titles.
- Allow Json files with the new naming convention used by Spotify: (`StreamingHistory_music_0.json`) etc.

### CODE REVIEW

File: spotistat.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def avg_day_load(df):
 
     ax2.axis('equal')
     ax2.pie(df_week_sum['noStreamsAvg'], labels=days, autopct='%1.2f%%')
-    ax2.set_title('Average Nubmer of Streams per Day')
+    ax2.set_title('Average Number of Streams per Day')
```
Comment (anthonio9) [inline spotistat.py:122]: Thanks, didn't see it on the last PR!

**Comment by anthonio9** [comment]
I'll add in the README, that files of naming format `StreamingHistory_music_x.json` are supported now, renaming is not necessary.

**Review by anthonio9 — APPROVED** [review]
Thanks for the fixes!

**Comment by anthonio9** [comment]
Thanks @danielbatchford, do you have any other suggestions for the improvements of the user experience? How did you find out about the project, was it useful?

**Comment by danielbatchford** [comment]
No worries. I checked out your repo as I wanted to get a quick summary of my listening habits, I can't think of any other improvements but the project is definitely useful!

**Pull request merged**